test/libsolidity/StandardCompiler.cpp

```diff
@@ -110,12 +110,12 @@ BOOST_AUTO_TEST_CASE(assume_object_input)
 
 	/// Use the string interface of StandardCompiler to trigger these
 	result = compile("");
-	BOOST_CHECK(containsError(result, "JSONError", "* Line 1, Column 1\n  Syntax error: value, object or array expected.\n"));
+	BOOST_CHECK(containsError(result, "JSONError", "* Line 1, Column 1\n  Syntax error: value, object or array expected.\n* Line 1, Column 1\n  A valid JSON document must be either an array or an object value.\n"));
 	result = compile("invalid");
-	BOOST_CHECK(containsError(result, "JSONError", "* Line 1, Column 1\n  Syntax error: value, object or array expected.\n"));
+	BOOST_CHECK(containsError(result, "JSONError","* Line 1, Column 1\n  Syntax error: value, object or array expected.\n* Line 1, Column 2\n  Extra non-whitespace after JSON value.\n"));
 	result = compile("\"invalid\"");
-	BOOST_CHECK(containsError(result, "JSONError", "Input is not a JSON object."));
-	BOOST_CHECK(!containsError(result, "JSONError", "* Line 1, Column 1\n  Syntax error: value, object or array expected.\n"));
+	BOOST_CHECK(containsError(result, "JSONError", "* Line 1, Column 1\n  A valid JSON document must be either an array or an object value.\n"));
+	BOOST_CHECK(!containsError(result, "JSONError","* Line 1, Column 1\n  Syntax error: value, object or array expected.\n"));
```

Comment:
Please also avoid changing the coding style in the lines above.

Reply:
Sorry.